Vagrantfile

```diff
@@ -0,0 +1,61 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# This script provisions a vagrant development environment with local
+# postgres and redis servers for development and testing.
+PROVISION = <<SCRIPT
+set -e
+
+# cd to /vagrant on login
+cd /vagrant
+grep -q 'cd /vagrant' ~/.bashrc || echo 'cd /vagrant' >> ~/.bashrc
+
+# Install system packages
+sudo apt-get update --quiet
+sudo apt-get install -y $(cat debian_packages.lst) postgresql
```

Comment:
@omarkhan Ok - I'm merge as is then, and let @brousch port this part of his PR, if he wants to.